docs/tutorial/part-six/index.md

```diff
@@ -1,31 +1,26 @@
 ---
-title: Transformer plugins
+title: ટ્રાન્સફોર્મર plugins
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) and [part 5](/tutorial/part-five/) before continuing here.
+> આ ટ્યુટોરીયલ Gatsby ડેટા સ્તર વિશેની શ્રેણીનો ભાગ છે. અહીં ચાલુ રાખતા પહેલા ખાતરી કરો કે [ભાગ ૪](/tutorial/part-four/) અને [ભાગ ૫](/tutorial/part-five/) પૂર્ણ કર્યો છે .
 
-## What's in this tutorial?
+## આ ટ્યુટોરિયલમાં શું છે?
 
-The previous tutorial showed how source plugins bring data _into_ Gatsby’s data system. In this tutorial, you'll learn how transformer plugins _transform_ the raw content brought by source plugins. The combination of source plugins and transformer plugins can handle all data sourcing and data transformation you might need when building a Gatsby site.
+અગાઉના ટ્યુટોરીયલ કેવી રીતે સ્રોત માહિતી _લાવવા_ plugins દર્શાવ્યું કે Gatsby ડેટા સિસ્ટમ છે. આ ટ્યુટોરીયલ માં, તમે જાણશો કે કેવી રીતે ટ્રાન્સફોર્મર plugins _પરિવર્તન_ કાચા સામગ્રી સ્રોત plugins દ્વારા લાવવામાં આવ્યા હતા. સોર્સ પ્લગિન્સ અને ટ્રાન્સફોર્મર પ્લગઇન્સનું સંયોજન Gatsby સાઇટ બનાવતી વખતે તમને જોઈતી બધી માહિતી સોર્સિંગ અને ડેટા ટ્રાન્સફોર્મેશનને નિયંત્રિત કરી શકે છે.
```

Comment:
`અગાઉના ટ્યુટોરીયલ કેવી રીતે સ્રોત માહિતી _લાવવા_ plugins દર્શાવ્યું કે` Please translate it properly. Don't translate `_into_`, `પ્લગઇન્સ` and ` _transform_`.

`કાચા સામગ્રી` This is confusing. Instead, use `raw content` in Gujarati.